src/Stryker.Core/Stryker.Core/Reporters/HtmlReporter/HtmlReporter.cs

```diff
@@ -38,6 +42,13 @@ public void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent)
             _consoleWriter.Write(Output.Green($"\nYour html report has been generated at: \n " +
                 $"file://{clickablePath} \n" +
                 $"You can open it in your browser of choice. \n"));
+
+            if (_options.OpenReporter == Options.Inputs.ReportType.HTMLReport)
```

Comment:
I don't think the Html reporter should be responsible for opening the report. Otherwise every openable report type will have to reimplement how to open the report.

Reply:
Each report has it's own way of opening right? Where would you suggest to put it?

Reply:
I only see two 'openable' report types, both of them are URI based.